lib/improver/nbhood/use_nbhood.py

```diff
@@ -36,6 +36,7 @@
 from improver.nbhood.nbhood import NeighbourhoodProcessing
 from improver.utilities.cube_checker import (
     check_cube_coordinates, find_dimension_coordinate_mismatch)
+from improver.blending.weights import WeightsUtilities
```

Comment:
The code only uses the static method "normalise". Can this be imported on its own?

Reply:
I wasn't sure, because it is a function within a class. I'll double check when I have some tests but I don't think its possible.

Reply:
I couldn't make this work.

Reply:
For later work - many of us seem to agree that WeightsUtilities shouldn't really be a class, as it doesn't implement any processing and contains only static methods which are quite general in terms of weighted blending.  So it might be worth having a new ticket to move these functions into a separate "utilities" module.